contacts/index_settings.json

```diff
@@ -153,6 +153,12 @@
                 "flow": {
                     "type": "keyword"
                 },
+                "flow_id": {
+                    "type": "integer"
+                },
```

Comment:
seems preferable to index a list of integers rather than UUIDs... and if we have to know about flow ids, why not switch `flow` over to being id based as well